SlicerCART/documentation/installation.md

```diff
@@ -0,0 +1,56 @@
+# SlicerCART Installation
+
+### Requirements
+
+* MacOS Sonoma or Sequoia is recommended 
+* A working version of [3D Slicer](https://download.slicer.org).
+  * N.B. The version currently used to develop this module is the version 5.6.2 (most stable release as of 2024-10-22). The version 5.2.2 has also been used.
```

Comment:
There is too much text, if you don't test it on many versions, just directly say the version
```suggestion
* A working version of [3D Slicer](https://download.slicer.org) (version 5.6.2).
```